deploy/vagrant/pip-req.txt

```diff
@@ -0,0 +1,6 @@
+paramiko
+PyYAML
+Jinja2
+httplib2
+ansible==1.9.1
+boto==2.34.0
```

Comment:
yes, libs before ansible is needed by ansible. boto is needed to interact with AWS.